js/apps/admin-ui/cypress/e2e/authentication_test.spec.ts

```diff
@@ -211,6 +211,8 @@ describe("Authentication test", () => {
   });
 
   it("Should display the default browser flow diagram", () => {
+    sidebarPage.goToRealm("master");
+    sidebarPage.goToAuthentication();
```

Comment:
Had to switch to master for this test because when orgs are enabled the default browser flow is changed for every new realm created, so the edges in the diagram connect different executions.

I've tried to add all new connections to the test, but we have two conditional executions in this flow and they have the same text, so cypress complains about multiple elements with the same text.

Using master for this test seemed to me the best way to get the issue fixed, and consequently all PRs unblocked. We can possibly have a separate issue to revisit this test and see if we can change it to take into account the changes to the browser flow introduced by the orgs which are now enabled by default.

Reply:
Logging an issue will do.

Reply:
https://github.com/keycloak/keycloak/issues/33077